lib/src/iterable_extensions.dart

```diff
@@ -166,6 +166,17 @@ extension IterableExtension<T> on Iterable<T> {
     }
   }
 
+  /// Returns an iterable containing only the non-`null` results
+  /// of applying the given [transform] function to each element in this iterable.
+  Iterable<R> mapNotNull<R extends Object>(R? Function(T) transform) sync* {
+    for (final element in this) {
```

Comment:
Just use `var` instead of `final`. This package generally doesn't use `final` for local variables.